### PR TITLE
docs(site): add documentation for "network" in manifest pages

### PR DESCRIPTION
**Pull request by efekarakus**
preview:
Screen Shot 2021-02-26 at 12 05 52 PM

_By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice._

### CODE REVIEW

File: site/content/docs/manifest/backend-service.md [hunk 1]
```diff
@@ -141,6 +145,26 @@ Scale up or down based on the average memory your service should maintain.
 
 <div class="separator"></div>
 
+<a id="network" href="#network" class="field">`network`</a> <span class="type">Map</span>    
+The `network` section contains parameters to connect to AWS resources in a VPC.
```
Comment (huanjani) [inline site/content/docs/manifest/backend-service.md]: nit: 
```suggestion
The `network` section contains parameters for connecting to AWS resources in a VPC.
```

File: site/content/docs/manifest/backend-service.md [hunk 2]
```diff
@@ -141,6 +145,26 @@ Scale up or down based on the average memory your service should maintain.
...
+<span class="parent-field">network.</span><a id="network-vpc" href="#network-vpc" class="field">`vpc`</a> <span class="type">Map</span>  
+Subnets and security groups attached to your tasks.
+
+<span class="parent-field">network.vpc.</span><a id="network-vpc-placement" href="#network-vpc-placement" class="field">`placement`</a> <span class="type">String</span>  
+Must be one of `'public'` or `'private'`. Defaults to launching your tasks in public subnets.
+
+!!! info inline end
+    Launching tasks in `'private'` subnets that need internet connectivity is only supported if you imported a VPC with
+    NAT Gateways when running `copilot env init`. See [#1959](https://github.com/aws/copilot-cli/issues/1959) for tracking
+    NAT Gateways support in Copilot generated VPCs.
```
Comment (huanjani) [inline site/content/docs/manifest/backend-service.md]: nit:
```suggestion
    NAT Gateways support in Copilot-generated VPCs.
```

File: site/content/docs/manifest/backend-service.md [hunk 3]
```diff
@@ -141,6 +145,26 @@ Scale up or down based on the average memory your service should maintain.
...
+
+<span class="parent-field">network.vpc.</span><a id="network-vpc-security-groups" href="#network-vpc-security-groups" class="field">`security_groups`</a> <span class="type">Array of String</span>
```
Comment (huanjani) [inline site/content/docs/manifest/backend-service.md]: nit:
```suggestion
<span class="parent-field">network.vpc.</span><a id="network-vpc-security-groups" href="#network-vpc-security-groups" class="field">`security_groups`</a> <span class="type">Array of Strings</span>  
```

File: site/content/docs/manifest/backend-service.md [hunk 4]
```diff
@@ -141,6 +145,26 @@ Scale up or down based on the average memory your service should maintain.
...
+Must be one of `'public'` or `'private'`. Defaults to launching your tasks in public subnets.
+
+!!! info inline end
+    Launching tasks in `'private'` subnets that need internet connectivity is only supported if you imported a VPC with
+    NAT Gateways when running `copilot env init`. See [#1959](https://github.com/aws/copilot-cli/issues/1959) for tracking
+    NAT Gateways support in Copilot generated VPCs.
+
+<span class="parent-field">network.vpc.</span><a id="network-vpc-security-groups" href="#network-vpc-security-groups" class="field">`security_groups`</a> <span class="type">Array of String</span>  
+Additional security group IDs associated with your tasks. Copilot always includes a security group for containers within your environment
+to communicate with each other.
```
Comment (huanjani) [inline site/content/docs/manifest/backend-service.md]: nit:
```suggestion
Additional security group IDs associated with your tasks. Copilot always includes a security group so containers within your environment
can communicate with each other.
```

File: site/content/docs/manifest/lb-web-service.md [hunk 5]
```diff
@@ -191,6 +190,26 @@ Scale up or down based on the service average response time.
...
+Must be one of `'public'` or `'private'`. Defaults to launching your tasks in public subnets.  
+
+!!! info inline end
+    Launching tasks in `'private'` subnets that need internet connectivity is only supported if you imported a VPC with 
+    NAT Gateways when running `copilot env init`. See [#1959](https://github.com/aws/copilot-cli/issues/1959) for tracking 
+    NAT Gateways support in Copilot generated VPCs.
+
+<span class="parent-field">network.vpc.</span><a id="network-vpc-security-groups" href="#network-vpc-security-groups" class="field">`security_groups`</a> <span class="type">Array of String</span>  
+Additional security group IDs associated with your tasks. Copilot always includes a security group for containers within your environment
+to communicate with each other.
```
Comment (huanjani) [inline site/content/docs/manifest/lb-web-service.md]: Change if changed above:
```suggestion
Additional security group IDs associated with your tasks. Copilot always includes a security group so containers within your environment
can communicate with each other.
```

File: site/content/docs/manifest/lb-web-service.md [hunk 6]
```diff
@@ -191,6 +190,26 @@ Scale up or down based on the service average response time.
 
 <div class="separator"></div>
 
+<a id="network" href="#network" class="field">`network`</a> <span class="type">Map</span>    
+The `network` section contains parameters to connect to AWS resources in a VPC.
```
Comment (huanjani) [inline site/content/docs/manifest/lb-web-service.md]: nit: change if changed above:
```suggestion
The `network` section contains parameters for connecting to AWS resources in a VPC.
```

File: site/content/docs/manifest/scheduled-job.md [hunk 7]
```diff
@@ -112,6 +112,26 @@ How long the job should run before it aborts and fails. You can use the units: `
 
 <div class="separator"></div>
 
+<a id="network" href="#network" class="field">`network`</a> <span class="type">Map</span>    
+The `network` section contains parameters to connect to AWS resources in a VPC.
```
Comment (huanjani) [inline site/content/docs/manifest/scheduled-job.md]: same as above:
```suggestion
The `network` section contains parameters for connecting to AWS resources in a VPC.
```

File: site/content/docs/manifest/scheduled-job.md [hunk 8]
```diff
@@ -112,6 +112,26 @@ How long the job should run before it aborts and fails. You can use the units: `
...
+Must be one of `'public'` or `'private'`. Defaults to launching your tasks in public subnets.
+
+!!! info inline end
+    Launching tasks in `'private'` subnets that need internet connectivity is only supported if you imported a VPC with
+    NAT Gateways when running `copilot env init`. See [#1959](https://github.com/aws/copilot-cli/issues/1959) for tracking
+    NAT Gateways support in Copilot generated VPCs.
+
+<span class="parent-field">network.vpc.</span><a id="network-vpc-security-groups" href="#network-vpc-security-groups" class="field">`security_groups`</a> <span class="type">Array of String</span>  
+Additional security group IDs associated with your tasks. Copilot always includes a security group for containers within your environment
+to communicate with each other.
```
Comment (huanjani) [inline site/content/docs/manifest/scheduled-job.md]: same as above:
```suggestion
Additional security group IDs associated with your tasks. Copilot always includes a security group so containers within your environment
can communicate with each other.
```

**Comment by efekarakus** [comment]
Oh god my docs are terrible 🤣

**Review by huanjani — APPROVED** [review]
Looks good!

**Comment by Lou1415926** [comment]
👀 good to me

**Review by Lou1415926 — APPROVED** [review]
:shipit:

**Pull request merged**